xml/System.Net.Security/SslApplicationProtocol.xml

```diff
@@ -32,7 +32,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Represents a values of TLS Application protocol.</summary>    
     <remarks>To be added.</remarks>
```

Comment:
should remark be updated as well

Reply:
`To be added.` is not rendered in the html docs. Unless we have something to remark I would leave like it is.